Services/Interfaces/web_interface/models/interface_settings.py

```diff
@@ -21,15 +21,16 @@
 
 def add_watched_symbol(symbol):
     watched_symbols = models.get_watched_symbols()
-    watched_symbols.append(symbol)
-    return _save_edition()
+    if symbol not in watched_symbols:
+        watched_symbols.append(symbol)
+        return _save_edition()
 
 
 def remove_watched_symbol(symbol):
     watched_symbols = models.get_watched_symbols()
     if symbol in watched_symbols:
```

Comment:
I think it would be better to try except ValueError instead of checking if symbol is in the watched_symbols list.

Reply:
👍